### ISSUE TITLE
Show furigana over kanji?

### ISSUE THREAD

**Issue by birtles** [birchill/10ten-ja-reader#2167]
From [AMO](https://addons.mozilla.org/en-US/firefox/addon/10ten-ja-reader/reviews/2148700/):

> One feature I wish it had is to display furigana over kanji when on hover.

I've deliberately avoided doing this in the past since I'm not sure of a really clean way to do this. For example, if the user hovers over 牧場 do we show the ぼくじょう reading or the まきば reading? Or do we show the kanji twice?

(I think I recall this topic being discussed in the context of WWWJDIC too with Jim Breen saying he too prefers to show the kanji and kana separately.)

**Comment by jonathan-s** [comment]
Leaving a link to an extension that does this exclusively for inspiration and for others thinking about wanting this.  
https://chromewebstore.google.com/detail/furiganator/ijfcmmkkmogaclaafpgcjdfcfibmhcfa

**Comment by birtles** [comment]
> Also issue duplicate here > [#975](https://github.com/birchill/10ten-ja-reader/issues/975)
> So those issues could be merged.

You're right. I misunderstood this issue. I thought it was about showing furigana over the kanji headwords in the words popup but since it says "when on hover" I guess they mean on the original page.

**Comment by birtles** [comment]
Closing this in favor of #975.
